simde/x86/sse2.h

```diff
@@ -5245,7 +5245,11 @@ simde_mm_pause (void) {
       __asm__ __volatile__("isb\n");
     #endif
   #elif defined(SIMDE_ARCH_POWER)
-    __asm__ __volatile__ ("or 27,27,27" ::: "memory");
+    #if defined(__APPLE__)
+      __asm__ __volatile__ ("or r27,r27,r27" ::: "memory");
+    #else
+      __asm__ __volatile__ ("or 27,27,27" ::: "memory");
+    #endif
```

Comment:
Good to know! If you are looking for other software that will also need patching this way, look at https://codesearch.debian.net/search?q=%22or+27%2C27%2C27%22&literal=1&perpkg=1